axelrod/tests/strategies/test_finite_state_machines.py

```diff
@@ -187,6 +203,19 @@ def test_all_states_reachable(self):
         extra_info = 'The following states are un-reachable: {}'.format(un_callable_states)
         self.assertEqual(un_callable_states, set(), msg=extra_info)
 
+    def test_strategy(self):
```

Comment:
should you mark this somehow as a regression test so that it doesn't get deleted in some later iteration?  or is there no need?